tv_apps/smart-system/js/browser_context_menu.js

```diff
@@ -299,11 +299,10 @@
 
   BrowserContextMenu.prototype._listAddAppToAppsItem = function(manifestURL) {
     return new Promise((resolve, reject) => {
-      var _ = navigator.mozL10n.get;
       if (!AppInstallManager.getAppAddedState(manifestURL)) {
         resolve({
           type: BUTTON_TYPE,
-          textRaw: _('add-to-apps'),
+          textL10nId: 'add-to-apps',
```

Comment:
Cool! Filed bug 1252579 and actually found a smarter way. In the place where the code adds the icon, I just create the `<span>` element and move text there.